### ISSUE TITLE
Composer error because of enshrined/svg-sanitize dependency

### ISSUE THREAD

**Issue by romainpoirier** [enovatedesign/craft-style-inliner#14]
I can't update Craft CMS from `4.4.4` to `4.4.5` using composer:

```
composer update
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires craftcms/cms 4.4.5 -> satisfiable by craftcms/cms[4.4.5].
    - You can only install one version of a package, so only one of these can be installed: enshrined/svg-sanitize[0.15.2, 0.15.3, 0.15.4, 0.16.0].
    - enovatedesign/craft-style-inliner 3.0.0 requires enshrined/svg-sanitize ^0.15.2 -> satisfiable by enshrined/svg-sanitize[0.15.2, 0.15.3, 0.15.4].
    - craftcms/cms 4.4.5 requires enshrined/svg-sanitize ~0.16.0 -> satisfiable by enshrined/svg-sanitize[0.16.0].
    - Root composer.json requires enovatedesign/craft-style-inliner 3.0.0 -> satisfiable by enovatedesign/craft-style-inliner[3.0.0].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

Removing `/vendor` + `composer.lock` + run `composer install` + run `composer update --with-all-dependencies` does not fix it.

As answered by the Craft CMS support [here](https://github.com/craftcms/cms/issues/12947):

> This is happening because a plugin you use: enovatedesign/craft-style-inliner requires "enshrined/svg-sanitize": "^0.15.2",, whereas Craft 4.4.5 has bumped the required version of enshrined/svg-sanitize to ~0.16.0 due to a security vulnerability in the previous versions.

Please, can you update the required version of `enshrined/svg-sanitize` in the latest version of `enovatedesign/craft-style-inliner`?

**Comment by jonleverrier** [comment]
+1

**Comment by Enovate-Jordan** [comment]
Hi all, I have published a new release `3.0.3` upgrading `enshrined/svg-sanitize` to `0.16.0` to satisfy the requirements for Craftcms version `4.5.0`.
